### ISSUE TITLE
Senua’s Saga: Hellblade II (2461850)

### ISSUE THREAD

**Issue by frozen-sea** [ValveSoftware/Proton#7754]
# Compatibility Report
- Name of the game with compatibility issues: Senua’s Saga: Hellblade II
- Steam AppID of the game: 2461850

## System Information
- GPU: RTX 4070 Ti
- Video driver version: Nvidia 550.78
- Kernel version: 6.9.1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/frozen-sea/0c65b6f8d516b7359882a8c031f143eb
- Proton version: 9.0-1

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.

[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15387462/steam-2461850.log)

## Symptoms
- Crash in graphics option menu

## Reproduction
- Go to Options -> Graphics, wait a few seconds

Switching to Proton Experimental solves this. Did not find any other obvious issues so far.

Edit: It doesn't happen on 8.0-5c, so this is a regression. Maybe just one that was already known and fixed?

**Comment by frozen-sea** [comment]
In addition to that, the game doesn't launch at all on Intel Arc. I don't see anything obvious in the log that would make it conclude DX12 isn't supported.

![Screenshot_20240522_024512](https://github.com/ValveSoftware/Proton/assets/89647969/5c0ff338-aeb8-44d6-a05b-8d6c722e0940)

GPU: Arc A580
Video driver version: Mesa 24.1
Kernel version: 6.8.7
Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/frozen-sea/b1d63ab4b85d2094de02ffeeb0cbf21c
Proton version: Experimental bleeding-edge

[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15396472/steam-2461850.log)

**Comment by JCDentonCore** [comment]
The game starts, I can play several minutes of the introduction but when it reaches a certain part before I can control the character, the image freezes and only the audio is heard.

I have to force the game to close since no error message appears, it just freezes.


Arch Linux kernel 6.9, Wayland, Mesa Dev 24.2
AMD RX 7900 XT

Proton experimental bleeding edge
Steam beta and beta runtime

Proton log:
[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15410037/steam-2461850.log)

System specs:
[specs.txt](https://github.com/ValveSoftware/Proton/files/15410043/specs.txt)

**Comment by simifor** [comment]
@JCDentonCore I couldn't reproduce the issue locally. Please try with the stock kernel and mesa, and try with proton 9. If the issue persists, upload new logs please.

**Comment by JCDentonCore** [comment]
@simifor 

> @JCDentonCore I couldn't reproduce the issue locally. Please try with the stock kernel and mesa, and try with proton 9. If the issue persists, upload new logs please.

No, same issue with proton 9 and kernel stock 6.9 from arch linux, mesa 24.2.0-devel(git-2740d92e3d)

[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15440660/steam-2461850.log)

![IMG_20240524_195223](https://github.com/ValveSoftware/Proton/assets/114034738/939de32f-55c2-4dfb-9c01-f0f61e689fd7)


Update test:

With mesa 24.1 stock, kernel 6.9 stock, proton 9, runtime stock, steam stock

Same issue, it just freezes at same point

[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15440861/steam-2461850.log)

![IMG_20240524_210059_edit_342633022309173](https://github.com/ValveSoftware/Proton/assets/114034738/9eeba1ef-d743-4e77-9378-624a51bb3b74)



Steam runtime log: 
[steamruntimelog.txt](https://github.com/ValveSoftware/Proton/files/15440947/steamruntimelog.txt)



PROTON 8.0-5c

Same issue

Log
[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15440968/steam-2461850.log)

**Comment by JCDentonCore** [comment]
@simifor With proton 9.0-2-rc1 same issue

Log from 9.0-2-rc1: 
[steam-2461850.log](https://github.com/ValveSoftware/Proton/files/15445402/steam-2461850.log)


Attached journald: 
[journald.txt](https://github.com/ValveSoftware/Proton/files/15445753/journald.txt)

I am using KDE 6.1 beta now, before I used version 6.0.4 stable, always under wayland, never try to start and test the game with Xorg


I'll try to try a session in xorg to see if there is a difference.

Next week I will finish building my new PC, keeping only the RX 7900 XT graphics, we will see if there are any changes, I will keep you informed, thanks for your hard work!

**Comment by urbenlegend** [comment]
Anyone else seeing weird black square artifacts particularly on NPC hair? It sometimes also appears on waterfalls.

Here's a video showing what I mean:

https://github.com/ValveSoftware/Proton/assets/241316/6360589e-6a80-429b-8c00-b8781484692d

It even seems to appear on Senua's face due to her tiny skin hairs that dot her face. It is less obvious than the previous example, but it still manifests as weird white shimmering all over her face: 

https://github.com/ValveSoftware/Proton/assets/241316/747be965-35aa-4a26-84d4-3ebdb2a2f965

**System Info**
Operating System: Arch Linux 
KDE Plasma Version: 6.0.4
KDE Frameworks Version: 6.2.0
Qt Version: 6.7.0
Kernel Version: 6.9.1-arch1-2 (64-bit)
Graphics Platform: Wayland
Processors: 24 × AMD Ryzen 9 3900X 12-Core Processor
Memory: 31.3 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3090/PCIe/SSE2
Product Name: X570 Taichi

**Comment by urbenlegend** [comment]
Another issue that I am experiencing is poor performance when playing Hellblade 2 after resuming from sleep. I get around 39fps compared to the usual 45-50fps when I play right after a fresh boot. This is reproducible every time with Nvidia 555.42.02 drivers with Kwin Wayland + Explicit sync patch. No other game seems to exhibit the same performance degradation. When this happens, GPU usage hovers arouind 75 percent instead of 97-99.

Steps to reproduce:
1. Restart computer
2. Play Hellblade 2
3. Note performance
4. Quit Hellblade 2
5. Suspend system
6. Play Hellblade 2 again
7. Notice a 5-10fps drop

**System Info**
Operating System: Arch Linux
KDE Plasma Version: 6.0.4
KDE Frameworks Version: 6.2.0
Qt Version: 6.7.0
Kernel Version: 6.9.1-arch1-2 (64-bit)
Graphics Platform: Wayland
Processors: 24 × AMD Ryzen 9 3900X 12-Core Processor
Memory: 31.3 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3090/PCIe/SSE2 w/ 555.42.02 drivers + Kwin Explicit Sync
Product Name: X570 Taichi

**Comment by runar-work** [comment]
I gave the intro a shot with RX 7600 and wasn't able to reproduce any freezes with that, at least, after testing Mesa 24.0.8 and 2740d92e3d, x11 and Wayland, 6.9.2-arch1-1 and 6.9.2-zen1-1-zen, and different graphics presets.

**Comment by JCDentonCore** [comment]
> Replying: https://github.com/ValveSoftware/Proton/issues/7754#issuecomment-2132070672

Maybe it's a problem with the nvidia drivers, in mesa 24.2 devel it doesn't happen when activating VSR but I do notice a slight flickering in certain parts of the hair, only in photo mode.

[steam-2461850.log](https://github.com/user-attachments/files/15524220/steam-2461850.log)


------------------------------------------------------------------------------------------------------------------------------------------------------------------------


What I do have is a serious problem with mesa 24.1, part of the mapping disappears, I updated mesa to the development version 24.2 devel and it was solved.

![20240531040059_1](https://github.com/ValveSoftware/Proton/assets/114034738/f1ab50ca-8bc4-46eb-bc50-38e135149073)

![20240531035732_1](https://github.com/ValveSoftware/Proton/assets/114034738/6774e1aa-0bc7-49d0-ac62-69169d9327af)

I have updated my PC and now I can play, the freezing error I had before was with a PC from thousands of years ago, so maybe it was a thing of the i3 processor or the low amount of RAM, although it is curious that the game always stops It would freeze at the same point and the game would run.

**Comment by kisak-valve** [comment]
Hello @JCDentonCore, friendly note that the Proton log you attached is not for this game.

**Comment by JCDentonCore** [comment]
https://github.com/ValveSoftware/Proton/issues/7754#issuecomment-2143445657
> Hello @JCDentonCore, friendly note that the Proton log you attached is not for this game.

Thanks for the heads up, fixed!
